.github/CONTRIBUTING.md

```diff
@@ -1,3 +1,42 @@
+# Product Roadmap
+
+The following is the planned Roadmap for the development of Kindly:  
+![Product Roadmap - updated Summer of 2024](./documentation_images/roadmap.png)
+
+All active Milestones in the Kindly project will follow this Roadmap and new feature suggestions will be considered for future releases.
+
+We encourage new contributors to work on any aspect of the project that feels appropriate to their experience and interest but please always consider where an Issue falls within the roadmap.
+
+If you submit a new Issue (whether it's a bug or new feature request) please indicate the most adequate Milestone based on the current Roadmap.
+
+## User Stories
+The following User Stories follow the projected Roadmap shown above.
+When working on any Issue collaborators are encouraged to look for additional improvement opportunities that will fullfil these stories.
+Similarly, new Issue submissions are always welcomed but may be prioritized for fulfillment if they conform to any of the stories below.
+
+### User Account Milestone 🗝️
+- As a User I want to have an option to reset my password from the login page
+- As a User I want to be able to suspend my account
+- As a User I want to be able to delete my account
+- As a User I want to be able to browse through Kindly before I need to create an account in order to interact Donations
+- As a User I want to be able to sign up and login to Kindly with my login from social platforms such as Google or Facebook
+- As a User I want to be able to keep my account logged in between visits to Kindly 
+
+### Donations Milestone 🤝
+- As a Donor I want to be able to mark any of my items as 'Reserved'
+- As a Donor I want to prevent new Inquiries about any Items I have marked as 'Reserved'
+- As a Refugee I want to receive confirmation what an Item has been reserved for me upon request
```

Comment:
This is potentially not our decision to make but I wonder about the use of the word Refugee here...it potentially excludes asylum seekers/ anyone who has not been officially granted refugee status for whatever reason?

Reply:
🏴 Uh, that's actually a good flag.

Throughout Kindly we use the word 'refugee' (for example right on the welcome page before signup) so I just kind of default to it in the assumption that came from the PO originally.

We should really just make sure that we're aligning everywhere with the language Trafalgar Girls uses to communicate with their community though - @Duncankatia is this something we should be adjusting on the platform and documentation?